models/public/densenet-121-tf/densenet-121-tf.md

```diff
@@ -2,64 +2,67 @@
 
 ## Use Case and High-Level Description
 
-This is a TensorFlow\* version of `densenet-121` model, one of the DenseNet\*
-group of models designed to perform image classification. The weights were converted from DenseNet-Keras Models. For details, see [repository](https://github.com/pudae/tensorflow-densenet/) and [paper](https://arxiv.org/abs/1608.06993).
+This is a TensorFlow\* version of `densenet-121` model, one of the DenseNet\* group of models designed to perform image classification.
+The weights were converted from DenseNet-Keras Models. For details, see [TensorFlow\* API docs](https://www.tensorflow.org/api_docs/python/tf/keras/applications/DenseNet121), [repository](https://github.com/tensorflow/tensorflow) and [paper](https://arxiv.org/abs/1608.06993).
```

Comment:
> The weights were converted from DenseNet-Keras Models.

This sentence no longer applies. These _are_ the DenseNet Keras models! :-)

Reply:
Removed